asset-transfer-private-data/application-javascript/app.js

```diff
@@ -221,11 +234,23 @@ async function main() {
             console.log('  result: ' + prettyJSONString(result.toString()));
 
             console.log('\n********* Demo deleting asset **************');
-            // Delete Asset2
-            console.log('--> Submit Transaction: DeleteAsset ' + assetID2);
-            statefulTxn = contractOrg1.createTransaction('DeleteAsset');
-
             let dataForDelete = { assetID: assetID2 };
+            try {
+                //Non-owner Org2 should not be able to DeleteAsset. Expect an error from DeleteAsset
+                console.log('--> Attempt Transaction: as Org2 DeleteAsset ' + assetID2);
+                statefulTxn = contractOrg2.createTransaction('DeleteAsset');
+                tmapData = Buffer.from(JSON.stringify(dataForDelete));
+                statefulTxn.setTransient({
+                    asset_delete: tmapData
+                });
+                result = await statefulTxn.submit();
```

Comment:
beyond the change in #309 , added this call to Delete, where error is expected and corresponding check in the chaincode